### PR TITLE
[10.x.x] Update Volume Framework toggle added to URP Asset

### PR DESCRIPTION
**Pull request by ellioman**
# Merge Status
Do not merge yet. I need Thomas to review this PR again before we can allow it to be merged.

# Purpose of this PR
The performance of updating the volume framework can be significant, especially on lower end platforms. URP currently always updates the framework every frame for each camera and users have asked for an option to disable them for projects that do not use volumes.

This PR does the following:
- Adds a `ResetVolumesToDefaultValues()` function to VolumeManager.cs
- Adds a "Update Volume Framework" toggle to the URP Asset under the advanced section
image
- Adds a Volume section in the camera inspector, moves the volume settings there and displays the update settings
image
- Adds a check which stops UniversalRenderPipeline.cs from updating volumes if the asset has the new toggle disabled
- Fixes typo "form" -> "from" in UniversalRenderPipelineCameraEditor.cs
- Fixes typo "valide" -> "valid" in UniversalRenderPipelineAsset.cs

# Comments to reviewers
We need to confirm that we see performance improvements with this PR as there might be other places in the volume system where this toggle might be needed. Such as Update() in Volume.cs from core. This still needs investigation.

# Yamato
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

### CODE REVIEW

**Comment by cinight** [comment]
Volume update is still running regardless of this checkbox setting when SceneView is visible <- is this intended design?

File: com.unity.render-pipelines.universal/Runtime/ScriptableRendererData.cs [hunk 1]
```diff
@@ -36,6 +37,12 @@ public List<ScriptableRendererFeature> rendererFeatures
             get => m_RendererFeatures;
         }
 
+        /// <summary>
+        /// Controls whether the volume framework should be updated every frame by
+        /// this renderer each frame or via scripting
```
Comment (martint-unity) [inline com.unity.render-pipelines.universal/Runtime/ScriptableRendererData.cs:42]: /// Controls whether the volume framework should be updated every frame by
        /// this renderer or via scripting

Comment (martint-unity) [reply com.unity.render-pipelines.universal/Runtime/ScriptableRendererData.cs:42]: Just looks better. Doenst need to fix in this PR

**Comment by ellioman** [comment]
> If my edits look good to you, then I approve.

Looks good

**Comment by ellioman** [comment]
This will be introduced alongside another PR for Volumes.

**Pull request closed without merging**